lib/diff_tools.py

```diff
@@ -275,9 +276,8 @@ def slice_list(lst, index, slice_len):
 
     try:
         local_dpsi = lst[int(left_bound):int(right_bound)]
+#         local_dpsi = lst[left_bound:right_bound]
```

Comment:
is this commented out code left here for a reason?

Reply:
Thanks I should remove this, the commented was original version. This line was changed to avoid TypeError.